native/cocos/core/scene-graph/Node.cpp

```diff
@@ -495,7 +495,7 @@ void Node::updateWorldTransformRecursive(uint32_t &dirtyBits) { // NOLINT(misc-n
             }
         }
     }
-    setDirtyFlag(static_cast<uint32_t>(TransformBit::NONE));
+    _transformFlags = (static_cast<uint32_t>(TransformBit::NONE));
```

Comment:
It seems `setDirtyFlag()` is not need? Can delete it?

Reply:
Already removed